<launch>
  
  <arg name="manager" value="nodelet_manager" />

  <node pkg="nodelet" type="nodelet" name="$(arg manager)" args="manager" output="screen" />

  <node pkg="nodelet" type="nodelet" name="ImageSubNodelet" 
          args="load rr_auto_dock/ImageSubNodelet $(arg manager)" output="screen" >
  </node>

  <node pkg="nodelet" type="nodelet" name="usb_cam"
           args="load usb_cam/UsbCamNodelet $(arg manager)" output="screen" >
      <param name="video_device" value="/dev/video-front" />
      <param name="camera_frame_id" value="front_camera_link" />
      <param name="image_width" value="1280" />
      <param name="image_height" value="720" />
      <param name="framerate" value="30" />
      <param name="pixel_format" value="mjpeg" />
      <param name="camera_info_url" value="package://rr_auto_dock/misc/ost.yaml" />
      <remap from="/usb_cam/image_raw" to="/camera"/>
      <remap from="/usb_cam/camera_info" to="/camera_info"/>
  </node>

</launch>